flygym/simulation.py

```diff
@@ -321,6 +321,10 @@ def close(self) -> None:
                 camera.output_path.parent.mkdir(parents=True, exist_ok=True)
                 camera.save_video(camera.output_path)
 
+        for fly in self.flies:
+            if hasattr(fly, "close"):
+                fly.close()
+
```

Comment:
Why would some flies not have the close method? Should we add an empty `.close()` to the fly base class then?